lib/standard_deviation.rb

```diff
@@ -1,5 +1,74 @@
-ages = [24, 30, 18, 20, 41]
+module Stats
 
-# Your code here for calculating the standard deviation
+    # Your code here for calculating the standard deviation
+    def sum_ages(ages)
+      ages.sum
+    end
 
-# When you find the standard deviation, print it out
+    def mean_ages(ages)
+      sum_ages(ages) / (ages.count.to_f)
+    end
+
+    def subtract_integers(ages)
+      subtracted = ages.map do |age|
+        (age - mean_ages(ages)).round(1)
+      end
+    end
+
+    def squared_integers(ages)
+      squared = subtract_integers(ages).map do |subtract|
+        (subtract ** 2).round(2)
+      end
+    end
+
+    def squared_sum(ages)
+      squared_integers(ages).sum
+    end
+
+    def divided_integers(ages)
+      squared_sum(ages) / ages.count
+    end
+
+    def standard_deviation(ages)
+      Math.sqrt(divided_integers(ages)).round(2)
+    end
```

Comment:
Interesting that you put this in a module! Thoughts behind that decision? It works, but I'm wondering the advantage to that over including it in your `event` class (of which there may be a few! I just want to make sure you're considering what they are! 😄)